solutions/factorial.js

```diff
@@ -0,0 +1,10 @@
+const factorial=(num,product=1)=>{
+  if (num==1){
+      return product;
+  }
+  else{
```

Comment:
You can shorten your code by getting rid of `else {product = product*num}` doing this: `return num*factorial()`